### DISCUSSION TITLE
현재 개발 진행상황

### DISCUSSION THREAD

**Discussion by paring-chan** [paring-chan/AdofaiUtils#2] [Announcements]
- [x] 자체 설정 UI 구현
- [ ] AdofaiUtils v1 기능
  - [x] CLS 키맵핑 추가
      - [x] 맵 입장
      - [x] 맵 정보
      - [x] 리로드
  - [x] 에디터 나가기
  - [ ] 1.0배 텍스트 숨기기

**Comment by patrick-choe** [comment]
와, 샌즈!

**Reply by paring-chan** [reply]
![image](https://user-images.githubusercontent.com/68010770/125801155-d635251d-a716-48ae-852f-f9ccb8cc3a64.png)

**Comment by wjdgustn** [comment]
누구세요

**Reply by paring-chan** [reply]
> 누구세요

바부 파랑이인데요

**Reply by wjdgustn** [reply]
?

**Comment by paring-chan** [comment]
\>ㅅ<

**Reply by patrick-choe** [reply]
파랑이 귀여워어

**Reply by paring-chan** [reply]
데헷 ☆
